forge/routes/api/settings.js

```diff
@@ -44,6 +44,9 @@ module.exports = async function (app) {
                 base_url: app.config.base_url,
                 license: app.license.status()
             }
+            if (app.config.features.enabled('customHostnames')) {
+                response.cnameTarget = app.config.driver.options?.customHostname?.cnameTarget
```

Comment:
Does this want to be a little more explicit that it's to do with custom hostnames?

Reply:
It's fine for now. We are free to move it later if we want.